test/trip_test.rb

```diff
@@ -41,5 +49,29 @@
         end.must_raise ArgumentError
       end
     end
+
+    it "raises an error because end time is before the start time" do
+      # if start_time > end_time #https://ruby-doc.org/core-2.6.3/Time.html
+      #   raise ArgumentError.new("Invalid, start time must be before end time")
+      # end
+      #what do we need: testing the initialize by creating a Trip object
+      @trip_data[:start_time] = Time.parse("2018-12-27 02:00:00 -0800")
+      @trip_data[:end_time] = Time.parse("2018-12-27 01:00:00 -0800")
```

Comment:
In your tests I'd recommend directly constructing objects instead of parsing strings.  Also, when using a `Time` or `Date` there are some helpful factory methods: `Time.now` and `Date.today`.

```suggestion
      @trip_data[:start_time] = Time.new(2018, 12, 27, 2)
      @trip_data[:end_time] = Time.new(2018, 12, 27,  1)
```